appservice/src/createAppService/SiteRuntimeStep.ts

```diff
@@ -53,8 +53,8 @@ export class SiteRuntimeStep extends AzureWizardPromptStep<IAppServiceWizardCont
     private getLinuxRuntimeStack(): ILinuxRuntimeStack[] {
         return [
             {
-                name: 'node|10.10',
-                displayName: 'Node.js 10.10 (LTS - Recommended for new apps)'
+                name: 'node|lts',
+                displayName: 'Node.js LTS (Recommended for new apps)'
```

Comment:
I'm not sure I'm ready to make this "Recommended for new apps" until we know exactly how it works. For example, what happens when Node updates LTS to a new major version? Do people's apps get updated automatically? If so, that doesn't sound great

@fiveisprime thoughts?